teuthology/orchestra/test/test_systemd.py

```diff
@@ -0,0 +1,56 @@
+import argparse
+import os
+
+from teuthology.orchestra import cluster
+from teuthology import misc
+from teuthology.orchestra.daemon.group import DaemonGroup
+import subprocess
+
+
+class FakeRemote(object):
+    pass
+
+
+def test_pid():
+    ctx = argparse.Namespace()
+    ctx.daemons = DaemonGroup(use_systemd=True)
+    remote = FakeRemote()
+
+    file_path = os.path.join(os.path.dirname(__file__),
+                             "files/daemon-systemdstate-pid-ps-ef.output")
+
+    with open(file_path) as ps:
+        ps_output = ps.read()
+    print("\nps -ef commond output: \n%s" % ps_output)
+
+    def sh(args):
```

Comment:
Since we are doing monkey patching here instead of providing pure patching, could you please add corresponding comment here.

Reply:
updated comment.